client/my-sites/hosting/main.js

```diff
@@ -174,7 +179,7 @@ class Hosting extends Component {
 		const getContent = () => {
 			const isGithubIntegrationEnabled = isAutomatticTeamMember( teams );
 			const WrapperComponent = isDisabled || isTransferring ? FeatureExample : Fragment;
-			const isStagingSiteEnabled = isEnabled( 'yolo/staging-sites-i1' );
+			const isStagingSiteEnabled = ! isStagingSite && isEnabled( 'yolo/staging-sites-i1' );
```

Comment:
Would it make more sense to add `isStagingSite` where we check for `isStagingSiteEnabled` before displaying the component? In another case, it can become weird if we remove the `isEnabled` check.

Reply:
Sure, did that with 7efdf46b80